pandas/tests/io/test_common.py

```diff
@@ -107,6 +107,26 @@ def test_iterator(self):
         tm.assert_frame_equal(first, expected.iloc[[0]])
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    @pytest.mark.parametrize('reader, module, error_class, fn_ext', [
+        (pd.read_csv, 'os', pd.compat.FileNotFoundError, 'csv'),
```

Comment:
can you make an issue to conform the error messages for feather, json to ``FileNotFound`` (rather than ``ValueError``). and fix msgpack for same.